data/json/effects.json

```diff
@@ -2073,6 +2073,52 @@
     "rating": "bad",
     "base_mods": { "vomit_chance": [ 500 ] }
   },
+  {
+    "type": "effect_type",
+    "id": "bile_stink",
+    "name": [ "Bile Stink" ],
+    "desc": [ "You're splattered with a greasy layer of liquified putrescence." ],
+    "apply_message": "You're covered in a foul smelling fluid!",
+    "rating": "bad",
+    "resist_traits": [ "FLOWERS", "ROSEBUDS", "SAPROVORE", "EATDEAD" ],
+    "base_mods": { "vomit_chance": [ 500, 2000 ] },
+    "enchantments": [
+      {
+        "values": [
+          { "value": "SOCIAL_PERSUADE", "add": -12 },
+          { "value": "SOCIAL_LIE", "add": -5 },
+          { "value": "SOCIAL_INTIMIDATE", "add": 10 },
+          { "value": "UGLINESS", "add": 8 },
```

Comment:
Unless this is physically disfiguring you this doesn't make sense. 
```suggestion
```

Reply:
It would make more sense with `bile_irritant` but only if visible, which is admittedly out of the scope of this PR.